cypress/e2e/annual-review/unpublish.feature

```diff
@@ -17,7 +19,7 @@ Feature:
     And I see "ANNUAL REVIEW OVERDUE"
     And the provider with reference "UNPUBLISH_DAY_TEST" should be unpublished
 
-
-
-
-
+  Scenario: A list item is deleted after being unpublished for a year
```

Comment:
Could we get another test actually - ensure list items that have been overdue for under a year aren't deleted?